effectivity/README.md

```diff
@@ -132,8 +142,11 @@ Use the Effectivity pattern when
 
 ## Consequences
 
-Effectivity pattern allows for simple temporal behaviour that relies on client awareness for 
-processing.
+Pros:
+* Allows for simple temporal behaviour.
+
+Cons:
+* Replies on client awareness of the temporality for processing.
```

Comment:
Did you mean to say "Relies on client awareness of the temporality for processing"?

Reply:
I did, thank you for catching that.